### ISSUE TITLE
Error: region not found for name: ap-southeast-3

### ISSUE THREAD

**Issue by markandersontrocme** [terraform-aws-modules/terraform-aws-pricing#5]
## Description

Tried using the module, copied the example from [here](https://github.com/terraform-aws-modules/terraform-aws-pricing/blob/v1.2.0/examples/pricing-resources/main.tf). When running `terraform plan` I get this error

```
│ Error: region not found for name: ap-southeast-3
│
│   with module.eks_cluster.module.pricing.data.aws_region.one["ap-southeast-3"],
│   on .terraform/modules/eks_cluster.pricing/modules/pricing/regions.tf line 24, in data "aws_region" "one":
│   24: data "aws_region" "one" {
│
╵
```

## Versions

 - Terraform: 1.1.1 (also tried with 0.15.4)

 - Provider(s): 
+ provider registry.terraform.io/hashicorp/aws v3.69.0
+ provider registry.terraform.io/terraform-aws-modules/http v2.4.1

 - Module:

## Reproduction

- use the module as shown [here](https://github.com/terraform-aws-modules/terraform-aws-pricing/blob/v1.2.0/examples/pricing-resources/main.tf)
- run `terraform plan`

### Code Snippet to Reproduce

```
module "pricing" {
  source  = "terraform-aws-modules/pricing/aws//modules/pricing"
  version = "1.2.0"

  debug_output = true

  resources = {
    "aws_instance.this" = {
      instanceType = "t3.large"
      location     = "us-east-1"
    }
  }
}

```
## Expected behavior

Looking to `pricing_per_resources`

## Actual behavior

Failed with an error

### Terminal Output Screenshot(s)

![image](https://user-images.githubusercontent.com/16004416/146573981-dba07552-1c96-4548-b870-84be889dde10.png)

## Additional context

I fixed it locally by manually editing the module, I will provide a PR with the change

**Comment by antonbabenko** [comment]
This issue has been resolved in version 1.3.0 :tada:

**Comment by antonbabenko** [comment]
Thanks @MarkAndersonTrocme for opening an issue and the PR!
